metapool/tests/test_metapool.py

```diff
@@ -190,6 +190,18 @@ def test_read_pico_csv_spectramax(self):
         with self.assertRaises(ValueError):
             read_pico_csv(fp_spectramax, plate_reader='foo')
 
+    def test_read_pico_csv_spectramax_negfix(self):
+        # Tests that Concentration values are clipped
+        # to a range of (0,60), eliminating possible
+        # negative concentration values.
+        fp_spectramax = os.path.join(os.path.dirname(__file__), 'data',
+                                     'pico_spectramax.txt')
```

Comment:
Thank you @RodolfoSalido; what do you think about adding a small test to make sure that there are actually values out of the range in fp_spectramax? I tried to find them but it was a bit complicated for me.